subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/DependencyInsightReportTask.java

```diff
@@ -687,4 +698,28 @@ public void renderNode(StyledTextOutput target, RenderableDependency node, boole
         }
     }
 
+    static class SerializableConfigurationData {
```

Comment:
💅 Inner classes here can be private and final.  Could use access modifiers on methods and constructors.